### PR TITLE
Add guide on Great Expectations Provider

### PR DESCRIPTION
**Pull request by petedejoy**
This PR adds a how-to guide on the shiny new official [Great Expectations Airflow Provider](https://github.com/great-expectations/airflow-provider-great-expectations).

@spbail - would love your feedback before merging!

### CODE REVIEW

File: guides/airflow-great-expectations.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: "Integrating Airflow and Great Expectations"
+description: "Using the Great Expectations provider natively in your Airflow DAGs."
+date: 2020-11-22T00:00:00.000Z
+slug: "airflow-great-expectations"
+tags: ["DAGs", "Integrations"]
+---
+
+## Overview
+
+[Great Expectations](https://greatexpectations.io) is an open source Python-based data validation framework. It allows you to test your data by expressing what you “expect” from it as simple declarative statements in Python, then run validation using those “expectations” against datasets. The [Great Expectations team maintains an Airflow provider](https://github.com/great-expectations/airflow-provider-great-expectations) that gives users a convenient method for running validation directly from their DAGs. This guide will walk you through the usage of the [official GreatExpectationsOperator](https://github.com/great-expectations/airflow-provider-great-expectations/blob/main/great_expectations_provider/operators/great_expectations.py_) and provide some guidance on how to configure an Airflow DAG containing a Great Expectations task to work with Airflow.
```
Comment (spbail) [inline guides/airflow-great-expectations.md:11]: There's an extra _ at the end of the second URL

File: guides/airflow-great-expectations.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+3. Created at least one Expectation Suite `my_suite`
+4. Optional: Created a Checkpoint `my_checkpoint`
+
+After setting up the project, you will see a `great_expectations` directory which contains several sub-directories, as well as the `great_expectations.yml` configuration file. Once that’s done, you can move on to invoking validation from Airflow, as described in the next section.
+
+## Using the Great Expectations Airflow Operator
+
+The [Great Expectations Airflow operator](https://github.com/great-expectations/airflow-provider-great-expectations) provides a convenient method for loading an existing Expectation Suite and using it to validate a batch of data. In order to use the operator in your DAG, follow these steps:
+
+1. Ensure that the great_expectations directory is accessible by your DAG. Ideally, it should be located in the same project as your DAG, but you can point the operator at any location.
```
Comment (spbail) [inline guides/airflow-great-expectations.md:38]: I just realized it probably helps to say something like "you can point the operator at any location by setting the `data_context_root_dir` parameter, more on that below"

File: guides/airflow-great-expectations.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    ```python
+    from great_expectations_provider.operators.great_expectations import GreatExpectationsOperator
+    ```
+
+4. Create a task using the `GreatExpectationsOperator`.
+
+The `GreatExpectationsOperator` supports multiple ways of invoking validation with Great Expectations:
+
+- Using an expectation suite name and `batch_kwargs`.
+- Using a list of expectation suite names and `batch_kwargs`.
```
Comment (spbail) [inline guides/airflow-great-expectations.md:57]: Just being nitpicky (and I was being messy in the doc), we usually uppercase Expectation Suites, Expectations, Data Context, and Checkpoints.

File: guides/airflow-great-expectations.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        'table': ‘my_table’,=
+        'datasource': ‘my_datasource’
+    },
+    dag=dag
+)
+```
+
+Note: If your `great_expectations` directory is not located in the same place as your DAG file, you will need to provide the `data_context_root_dir` parameter.
+
+By default, a Great Expectations task will run validation and raise an Airflow Exception if any of the tests fails. To override this behavior and continue running even if tests fail, set the `fail_task_on_validation_failure` flag to “False”.
```
Comment (spbail) [inline guides/airflow-great-expectations.md:76]: Also my fault, I'd actually write this as `AirflowExpectation` since it's the actual exception name.

**Review by spbail — COMMENTED** [review]
Just basically nitpicking my own spelling haha. Looks good otherwise!

File: guides/airflow-great-expectations.md (hunk 4 above)
Comment (petedejoy) [reply guides/airflow-great-expectations.md:76]: Assuming you mean `AirflowException` here. Jargon overload :)

File: guides/airflow-great-expectations.md (hunk 1 above)
Comment (petedejoy) [reply guides/airflow-great-expectations.md]: Good catch!

**Review by ryw — APPROVED** [review]
Looks good.

Merge it in, and we can continue to edit it as necessarily with smaller PRs/commits.

**Pull request merged**